sdk/storage/storage-queue/test/utils/recorder.ts

```diff
@@ -153,9 +153,9 @@ abstract class Recorder {
     return updatedRecording;
   }
 
-  public skip(): boolean {
-    return skip.includes(this.filepath);
-  }
+  // public skip(): boolean {
```

Comment:
Do we still need commentted code here?

Reply:
No, this will be deleted.
This current PR is just to show how the skip list can be avoided and hear some feedback for it.
Instead of doing the updates in recorder.ts files, I'm planning to update the new common recorder https://github.com/Azure/azure-sdk-for-js/pull/4281.